test/controllers/works_controller_test.rb

```diff
@@ -22,95 +38,199 @@
 
   describe "index" do
     it "succeeds when there are works" do
```

Comment:
You don't have any tests for guest users.  A good test for a guest user would look like:

```ruby
  get works_path

  flash[:error].must_equal "You must be logged in to see this page"
  must_respond_with :redirect
```